backend/src/main/java/de/wordsmith/backend/controller/WordController.java

```diff
@@ -0,0 +1,30 @@
+package de.wordsmith.backend.controller;
+
+import de.wordsmith.backend.model.Word;
+import de.wordsmith.backend.service.PonsApiService;
+import de.wordsmith.backend.service.WordflowService;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/word")
+public class WordController {
+
+    private final WordflowService wordflowService;
+    private final PonsApiService ponsApiService;
+
+    public WordController(WordflowService wordflowService, PonsApiService ponsApiService) {
+        this.wordflowService = wordflowService;
+        this.ponsApiService = ponsApiService;
+    }
```

Comment:
when working with Lombok you can substitute these lines with `@RequiredArgsConstructor` at your class

Reply:
Thanks for the tip! Now implemented in both the controller and service class